lib/fluent/plugin/kubernetes_metadata_common.rb

```diff
@@ -54,10 +54,28 @@ def parse_pod_metadata(pod_object)
         self.de_dot!(labels)
         self.de_dot!(annotations)
       end
+
+      # collect container informations
+      container_meta = {}
+      begin
+        pod_object['status']['containerStatuses'].each do|container_status|
+          # get plain container id (eg. docker://hash -> hash)
+          container_id = container_status['containerID'].sub /^[-_a-zA-Z0-9]+:\/\//, ''
+          container_meta[container_id] = {
+              'name' => container_status['name'],
+              'image' => container_status['image'],
+              'image_id' => container_status['imageID']
+          }
+        end
+      rescue
+        log.debug("parsing container meta information failed for: #{pod_object['metadata']['namespace']}/#{pod_object['metadata']['name']} ")
```

Comment:
search this code base I believe `log` has an on_debug method we should impl here like:
```
log.on_debug do
  log.debug(msg)
end
```